.github/workflows/php-js-e2e-tests.yml

```diff
@@ -64,15 +78,28 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - uses: actions/checkout@v3
+
+            - name: Cache node_modules
+              id: cache-node-modules
+              uses: actions/cache@v3
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: node_modules
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+
             - name: Setup node version and npm cache
               uses: actions/setup-node@v3
               with:
                   node-version-file: '.nvmrc'
                   cache: 'npm'
-            - name: Npm install and build
-              run: |
-                  npm ci --no-optional
-                  FORCE_REDUCED_MOTION=true npm run build:e2e-test
+
+            - name: Install Node Dependencies
+              if: steps.cache-node-modules.outputs.cache-hit != 'true'
+              run: npm ci --no-optional
```

Comment:
We've already run the `Install Node Dependencies` step in the first job (e.g.; the `JSE2EWithGutenberg` job), I think it's safe to remove this step from this one.

Reply:
See [comment above](https://github.com/woocommerce/woocommerce-blocks/pull/6404/files/6bd2555e729593a171b59d1a6b3c2037560481cf#r880346473)